### DISCUSSION TITLE
Theme not working for top langauge card

### DISCUSSION THREAD

**Discussion by AdwaitKulkarni58** [anuraghazra/github-readme-stats#1847] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
The problem currently is that there is support for themes for the other cards, but not the top languages card, so it doesn't look uniform on the profile.

**Describe the solution you'd like**
Please add the same themes you have for the other cards to the top languages card as well.

**Comment by rickstaa** [comment, marked as answer]
@AdwaitKulkarni58 Thanks for your question. We do support themes for the top language card. The correct syntax is:

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra&theme=radical)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra&theme=radical)](https://github.com/anuraghazra/github-readme-stats)
